.actor/Dockerfile

```
@@ -1,30 +1,24 @@
-# First, specify the base Docker image.
-# You can see the Docker images from Apify at https://hub.docker.com/r/apify/.
-# You can also use any other image from Docker Hub.
 FROM apify/actor-python:3.11
 
-# Second, copy just requirements.txt into the actor image,
-# since it should be the only file that affects the dependency install in the next step,
-# in order to speed up the build
-COPY requirements.txt ./
+RUN rm -rf /usr/src/app/*
+WORKDIR /usr/src/app
+
+COPY pyproject.toml /.
+COPY poetry.toml /.
+COPY poetry.lock /.
```

Not sure if you're copying them to root on purpose, shouldn't you be copying them to the workdir?
```suggestion
COPY pyproject.toml ./
COPY poetry.toml ./
COPY poetry.lock ./
```